### ISSUE TITLE
Have you reproduced similar results as original paper?

### ISSUE THREAD

**Issue by chenbinghui1** [lyc0929/OOTDiffusion-train#2]

**Comment by lyc0929** [comment]
I have been training with a single  A100 for four days and have achieved significant results, which makes me convinced that the code works.In addition, I sent you an email, please check it.

**Comment by siqi0905** [comment]
I'm also curious about how you reproduced the same results as the original paper. Could you share more training details? Also, I'm wondering if training only the main UNet is feasible.

**Comment by xuyu666** [comment]
Hello, author
I'd also like to see the results of your reproduction, can you send me a copy of my e-mail, my e-mail is 2622960263@qq.com

**Comment by ManuelCecere** [comment]
Hello author, would you be available to share your results with me too? My email is manucp44@gmail.com
I was wondering about the loss function you wrote: 
`                loss = F.mse_loss(noise_pred.float(), noise.float(), reduction="mean")
`

since it seems that noise is defined as `torch.randn_like(latents)` so I don't get how the model is learning, could you help me understand that?

**Comment by appleyang123** [comment]
I use your code. But after training 40k steps, I have tested model on test data. The results are here:
<img width="892" alt="image" src="https://github.com/lyc0929/OOTDiffusion-train/assets/166299531/d3cedb20-603e-45db-8e03-cda8b22291dc">

**Comment by orshi** [comment]
> I have been training with a single A100 for four days and have achieved significant results, which makes me convinced that the code works.In addition, I sent you an email, please check it.
 also expected to see the results of your reproduction it_xie@hotmail.com

**Comment by ruanruano** [comment]
> I'm also curious about how you reproduced the same results as the original paper. Could you share more training details? Also, I'm wondering if training only the main UNet is feasible.

I have the same question, have you trained it? No matter if it is a configuration like the paper or just training the main unet
